src/SFA.DAS.PensionsRegulatorApi.UnitTests/Application/Queries/GetValidatedOrganisations/Given_Repository_Returns_Results.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture;
+using FluentAssertions;
+using MediatR;
+using NSubstitute;
+using NUnit.Framework;
+using PensionsRegulatorApi.Application.Queries;
+using PensionsRegulatorApi.Data;
+using PensionsRegulatorApi.Domain;
+
+namespace SFA.DAS.PensionsRegulatorApi.UnitTests.Application.Queries.GetValidatedOrganisations
+{
+    [TestFixture]
+    [ExcludeFromCodeCoverage]
+    public class Given_Repository_Returns_Results
+    {
+        private IOrganisationRepository _mockRepository;
+        private IRequestHandler<global::PensionsRegulatorApi.Application.Queries.GetValidatedOrganisations, IEnumerable<Organisation>> _sut;
+        private IEnumerable<Organisation> _repositoryResults;
+
+        public Given_Repository_Returns_Results()
```

Comment:
Can we make the test names less implementation specific?  We need to test the expected behaviour not how we're doing it.